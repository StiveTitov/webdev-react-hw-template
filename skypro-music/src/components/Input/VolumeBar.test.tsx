import renderer from "react-test-renderer";
import { VolumeBar } from ".";

describe('Компанент- полосы регулировки звука', () => {
    it('корректно отрисовывает разметку с начальными значениями', () => {
      // Тестовый код
      const component = renderer.create(<VolumeBar volume={50} handleVolume={10} />).toJSON();
  expect(component).toMatchSnapshot();
    });
   
    
   });