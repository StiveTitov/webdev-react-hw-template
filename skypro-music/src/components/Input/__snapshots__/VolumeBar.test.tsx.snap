// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компанент- полосы регулировки звука корректно отрисовывает разметку с начальными значениями 1`] = `
<input
  className="barVolumeProgress"
  max={100}
  min={0}
  onChange={10}
  type="range"
  value={50}
/>
`;
